main.css

```diff
@@ -0,0 +1,729 @@
+  height: 16rem;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: top center;
+  position: relative;
+}
+
+.apps1 {
+  color: red;
+  text-rendering: optimizeLegibility;
+  font-size: 3rem;
+  vertical-align: baseline;
+  font-weight: bolder;
+}
+
+.small-flex {
+  display: flex;
+  justify-content: space-between;
+  text-decoration: none;
+  padding: 0.5rem;
+}
+
+.small-flex11 ul {
+  background-color: rgb(248, 250, 222);
+  font-size: 1.2rem;
+  padding: 2rem;
+  list-style-type: disc;
+}
+
+.grid-last {
+  position: relative;
+  background-image: url("images/28.jpg");
+}
+
+.grid-last2 {
+  position: relative;
+  background-image: url("images/51.jpg");
+}
+
+.grid-last3 {
+  position: relative;
+  background-image: url("images/52.jpg");
+}
+
+.grid-last4 {
+  position: relative;
+  background-image: url("images/53.jpg");
+}
+
+.grid-last5 {
+  position: relative;
+  background-image: url("images/54.jpg");
+}
+
+.grid-last6 {
+  position: relative;
+  background-image: url("images/55.jpg");
+}
+
+.grid-last7 {
+  position: relative;
+  background-image: url("images/57.jpg");
+}
+
+.story-dealDiscount {
+  background-image: url("https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/deals-ribbon.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  color: #000;
+  display: block;
+  height: 80px;
+  overflow: hidden;
+  pointer-events: none;
+  position: absolute;
+  right: 0;
+  top: 0;
+  width: 80px;
+  z-index: 17;
+}
+
+.story-dealDiscount div {
+  transform: rotate(45deg);
+  font-size: 14px;
+  left: 8px;
+  text-align: center;
+  text-transform: uppercase;
+  top: 20px;
+  width: 100%;
+  position: absolute;
+  z-index: 27;
+  font-weight: bolder;
+  text-rendering: optimizeLegibility;
+}
+
+.footer {
+  width: 100%;
+  height: 30rem;
+}
+
+.first-foot {
+  width: 100%;
+  height: 20rem;
+}
+
+.inner-foot1 {
+  height: 10rem;
+  background-color: #131313;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.last-footer {
+  color: #808080;
+  display: flex;
+  justify-content: space-around;
+  text-align: center;
+  list-style-type: none;
+  padding: 0;
+}
+
+.circlar-li {
+  background: rgba(128, 128, 128, 0.287);
+  border-radius: 50%;
+  height: 40px;
+  padding: 12px;
+  text-align: center;
+  width: 3rem;
+  margin-right: 1rem;
+}
+
+.last-footer1 {
+  display: flex;
+  justify-content: space-around;
+  margin: 0.6rem;
+  color: #9f9f9f;
+  list-style-type: none;
+}
+
+.last-footer1 li a:hover {
+  color: white;
+}
+
+.inner-foot22 {
+  height: 4rem;
+  width: 22rem;
+  display: block;
+  justify-content: center;
+}
+
+.foot55 {
+  color: white;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: black;
+}
+
+.circlar-li:hover {
+  background-color: #9f9f9f;
+}
+
+@media only screen and (max-width: 700px) {
+  .contain-nav-div {
+    display: block;
+    justify-content: space-between;
+  }
+
```

Comment:
You only have a media query for the small screen, you need for the medium screen as well. The medium screen entirely looks nothing like the original page. Kindly fix.

**CSS styles should use media queries for 3 breakpoints** 
* small: max-width around 700
* medium: max-width around 1000
* large: mwidth over 1000